src/vue/forms/IssuanceForm.vue

```diff
@@ -0,0 +1,234 @@
+            type="number"
+            v-model="form.amount"
+            @blur="touchField('form.amount')"
+            id="issuance-amount"
+            :label="'issuance.amount-lbl' | globalize"
+            :error-message="getFieldErrorMessage(
+              'form.amount',
+              { from: 0.000001, to: availableAmount.value }
+            )"
+          />
+          <p
+            v-if="availableAmount"
+            class="issuance-form__amount-label"
+          >
+            {{ availableAmount.currency }}
+          </p>
+        </div>
+        <p
+          v-if="availableAmount.value"
+          class="issuance-form__available-amount-hint"
+        >
+          {{ 'issuance.available-for-issuance-lbl' | globalize }}
+          {{ availableAmount | formatMoney }}
+        </p>
+      </div>
+    </div>
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="form.receiver"
+          @blur="touchField('form.receiver')"
+          id="issuance-receiver"
+          :label="'issuance.receiver-lbl' | globalize"
+          :error-message="getFieldErrorMessage('form.receiver')"
+        />
+      </div>
+    </div>
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="form.reference"
+          @blur="touchField('form.reference')"
+          id="issuance-reference"
+          :error-message="getFieldErrorMessage('form.reference')"
+          :label="'issuance.reference-lbl' | globalize"
+        />
+      </div>
+    </div>
+    <div class="app__form-actions">
+      <button
+        v-ripple
+        type="submit"
+        class="issuance-form__submit-btn"
+        :disabled="formMixin.isDisabled"
+      >
+        {{ 'issuance.issue-btn' | globalize }}
+      </button>
+      <button
+        v-ripple
+        type="button"
+        class="issuance-form__cancel-btn"
+        :disabled="formMixin.isDisabled"
+        @click.prevent="cancelForm"
+      >
+        {{ 'issuance.cancel-btn' | globalize }}
+      </button>
+    </div>
+  </form>
+</template>
+
+<script>
+import IssuanceFormMixin from '@/vue/mixins/issuance-form.mixin'
+
+import { Bus } from '@/js/helpers/event-bus'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+import { Sdk } from '@/sdk'
+import { base } from '@tokend/js-sdk'
+
+import { required, amountRange } from '@validators'
+
+export default {
+  name: 'issuance-form',
+  mixins: [IssuanceFormMixin],
+  data: _ => ({
+    form: {
+      asset: '',
+      amount: 0,
+      receiver: '',
+      reference: ''
+    }
+  }),
+  validations () {
+    return {
+      form: {
+        asset: { required },
+        amount: {
+          required,
+          amountRange: amountRange(
+            0.000001,
+            this.availableAmount ? this.availableAmount.value : 0
+          )
+        },
+        receiver: { required },
```

Comment:
let’s add `emailOrAccountId` global validator. we definitely will use it later in the future forms. the base already contain account id validation tool, and email is a regexp.

Reply:
Resolved, added